pkg/blobfuse-proxy/main.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"os/exec"
+	"strings"
+
+	"google.golang.org/grpc"
+	mount_azure_blob "sigs.k8s.io/blob-csi-driver/pkg/blobfuse-proxy/pb"
+)
+
+func init() {
+	_ = flag.Set("logtostderr", "true")
+}
+
+var (
+	blobfuseProxyEndpoint = flag.String("blobfuse-proxy-endpoint", "unix://tmp/blobfuse-proxy.sock", "blobfuse-proxy endpoint")
+)
+
+type MountServer struct {
+	mount_azure_blob.UnimplementedMountServiceServer
+}
+
+// NewMountServer returns a new Mountserver
+func NewMountServiceServer() *MountServer {
+	return &MountServer{}
+}
+
+// MountAzureBlob mounts an azure blob container to given location
+func (server *MountServer) MountAzureBlob(ctx context.Context,
+	req *mount_azure_blob.MountAzureBlobRequest,
+) (resp *mount_azure_blob.MountAzureBlobResponse, err error) {
+
+	args := req.GetMountArgs()
+	log.Printf("received mount request: Mounting with args %v \n", args)
+
+	var result mount_azure_blob.MountAzureBlobResponse
+	cmd := exec.Command("blobfuse", strings.Split(args, " ")...)
+	cmd.Env = append(os.Environ(), "AZURE_STORAGE_ACCOUNT="+req.GetAccountName())
+	cmd.Env = append(cmd.Env, "AZURE_STORAGE_ACCESS_KEY="+req.GetAccountKey())
```

Comment:
write like this:
https://github.com/kubernetes-sigs/blob-csi-driver/blob/21c745910ada964f5c8366812d91ebeca3b5abfd/pkg/blob/nodeserver.go#L216-L220